sass/cd/cd-layout/_cd-layout.scss

```diff
@@ -35,15 +35,24 @@ body {
   }
 
   // Grid is used to fix the footer to the bottom of the screen, regardless of page min-height.
+  // Header height is fixed.
+  // Highlighted should contain Drupal messages and will be present even if empty.
```

Comment:
I checked a few implementations, GHO as well, and I can see the highlighted region div render despite no visible content. I think this is fine, and we can use this region to hold the Drupal messages - this might be standard Drupal.

This means a potential configuration change is also needed when updating the implementations, to move the status messages to the correct region.